### PR TITLE
(maint) Fix execute_command parameter passing

### PR DESCRIPTION
**Pull request by Filipovici-Andrei**
Fixed how parameters are passed from Facter::Core::Execution to Facter::Core::Execution::Base

### CODE REVIEW

**Comment by Filipovici-Andrei** [comment]
jenkins please test this on all

File: lib/facter/custom_facts/util/resolution.rb [hunk 1]
```diff
@@ -139,7 +139,7 @@ def options(options)
       def setcode(string = nil, &block)
         if string
           @code = proc do
-            output = Facter::Core::Execution.execute(string, on_fail: nil)
+            output = Facter::Core::Execution.execute_command(string)
```
Comment (IrimieBogdan) [inline lib/facter/custom_facts/util/resolution.rb:142]: Maybe this should be keep as `Facter::Core::Execution.execute(string, on_fail: nil)` as it expands commands.

Comment (Filipovici-Andrei) [reply lib/facter/custom_facts/util/resolution.rb]: reverted the change

**Comment by Filipovici-Andrei** [comment]
jenkins please test this on all

**Comment by Filipovici-Andrei** [comment]
jenkins please test this on all

**Review by IrimieBogdan — APPROVED** [review]

**Pull request merged**